src/lib/jsonParseV2/parseSubscription.cpp

```diff
@@ -459,6 +459,39 @@ static std::string parseMqttTopic(ConnectionInfo* ciP, SubscriptionUpdate* subsP
 
 
 
+/* ****************************************************************************
+*
+* parseHttpTimeout -
+*/
+static std::string parseHttpTimeout(ConnectionInfo* ciP, SubscriptionUpdate* subsP, const Value& http)
+{
+  Opt<int64_t> httpTimeoutOpt = getInt64Opt(http, "httpTimeout");
+  if (!httpTimeoutOpt.ok())
+  {
+    return badInput(ciP, httpTimeoutOpt.error);
+  }
+  if (httpTimeoutOpt.given)
+  {
+    if ((httpTimeoutOpt.value < 0) || (httpTimeoutOpt.value > MAX_SUB_HTTP_TIMEOUT))
+    {
+      // FIXME - There is any way to display the value of the constant MAX_SUB_HTTP_TIMEOUT? 
+      return badInput(ciP, "httpTimeout field must be an integer between 0 and MAX_SUB_HTTP_TIMEOUT");
```

Comment:
Maybe this (unsure, check it):

```suggestion
      return badInput(ciP, "httpTimeout field must be an integer between 0 and " + std::string(MAX_SUB_HTTP_TIMEOUT));
```

Reply:
Used std::to_string() instead of std::string()